SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1764,8 +1765,12 @@ void CSolver::AdaptCFLNumber(CGeometry **geometry,
 
     /* Check if we should decrease or if we can increase, the 20% is to avoid flip-flopping. */
     resetCFL = linRes > 0.99;
-    reduceCFL = linRes > 1.2*linTol;
-    canIncrease = linRes < linTol;
+    unsigned long iter = config->GetMultizone_Problem() ? config->GetOuterIter() : config->GetInnerIter();
+
+    /* only change CFL number when larger than starting iteration */
+    reduceCFL = linRes > 1.2*linTol && (iter > startingIter);
+
+    canIncrease = (linRes < linTol) && (iter > startingIter);
```

Comment:
We should use by default start at iteration 0 or else all existing adaptive CFL will have different residuals.

```suggestion
    reduceCFL = linRes > 1.2*linTol && (iter >= startingIter);

    canIncrease = (linRes < linTol) && (iter >= startingIter);
```